testing-with-jest.test.js

```diff
@@ -8,20 +8,38 @@
 
 // --------------------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
 
+describe("addThemUp", () => {
+  test("function called addThemUp that takes two numbers as arguments and returns the sum", () => {
+    expect(addThemUp(2, 5)).toEqual(7)
+  })
+})
```

Comment:
Great! I'd want to encourage you to write additional expects to check for more edge cases! What if the function was passed something that was not integer datatypes?